gsoc.yml

```diff
@@ -21,7 +21,7 @@ gsoc2018:
     rss_feed: "https://medium.com/feed/@swapsha96"
     project: 'TimeLab'
   prakharcode:
-    rss_feed: "FIXME"
+    rss_feed: "https://prakharcode.github.io/feed.xml"
```

Comment:
@prakharcode would it be possible to get the feed only for the GSoC tag? You may want to write about other stuff in your blog.